R/odbc.R

```diff
@@ -1,9 +1,24 @@
-#' @aliases NULL
-#' @aliases odbc-package
-#' @importFrom Rcpp sourceCpp
-#' @importFrom blob blob
-#' @importFrom bit64 integer64
-#' @importFrom hms hms
-#' @useDynLib odbc, .registration = TRUE
-#' @keywords internal
-"_PACKAGE"
+#' Set the Transaction Isolation Level for a Connection
+#'
+#' @param levels One or more of \Sexpr[stage=render, results=rd]{odbc:::choices_rd(names(odbc:::transactionLevels()))}.
+#' @inheritParams DBI::dbDisconnect
+#' @seealso \url{https://docs.microsoft.com/en-us/sql/odbc/reference/develop-app/setting-the-transaction-isolation-level}
+#' @export
+#' @noMd
+#' @examples
+#' \dontrun{
+#' # Can use spaces or underscores in between words.
+#' odbcSetTransactionIsolationLevel(con, "read uncommitted")
+#'
+#' # Can also use the full constant name.
+#' odbcSetTransactionIsolationLevel(con, "SQL_TXN_READ_UNCOMMITTED")
+#' }
+odbcSetTransactionIsolationLevel <- function(conn, levels) {
```

Comment:
This seems like unexpected content for the `odbc.R` file.

Reply:
I'm thinking we'll accumulate random `odbc...` functions in this file.